blackbox 2.0.0 fails to build

### Describe the bug
blackbox-2.0.0 fails to build with test failures.

### Steps To Reproduce
Steps to reproduce the behavior:
1. nix-env -i blackbox

### Expected behavior
blackbox builds

### Screenshots
```
  make copy-uninstall     Remove blackbox files from /nix/store/905xlz3bx4j1yycbd8fj7f617av1p0la-blackbox-2.0.0/bin/
  make test               Run tests
running tests
check flags: SHELL=/nix/store/715y5lnw12vdcc3hh9sc9vbkdi6c83x6-bash-4.4-p23/bin/bash VERBOSE=y test
/nix/store/715y5lnw12vdcc3hh9sc9vbkdi6c83x6-bash-4.4-p23/bin/bash: pkill: command not found
tools/auto_system_test: line 7: spawn: command not found
tools/auto_system_test: line 25: syntax error near unexpected token `}'
tools/auto_system_test: line 25: `}'
make: *** [Makefile:145: confidence] Error 2
builder for '/nix/store/ik7dqwnsrx3p59x94cdngbmsqwajc7gx-blackbox-2.0.0.drv' failed with exit code 2
```

### Notify maintainers
@ericsagnes 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-darwin"`
 - host os: `Darwin 20.5.0, macOS 10.16`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.3.15`
 - channels(paikens): `"home-manager, nixos-21.05.2501.94c989365d5, nixpkgs-21.11pre309233.70dc433f2eb, nixpkgs-master, nur-duckpuppy, nur-gerschtli, unstable-21.11pre309163.23485f23ff8, nixpkgs-unstable-21.11pre299378.db6e089456c, darwin"`
 - nixpkgs: `/nix/store/jygrjgj2q8qbnhza762pfxwqiqgm1yla-nixpkgs-unstable.tar.gz`
```

fails on aarch64-linux but for a different reason
```
blackbox> ========== Decrypting new/changed files: START
blackbox> Please enter the passphrase to unlock the OpenPGP secret key:
blackbox> "Bob Example (my password is the lowercase letter b) <bob@example.com>"
blackbox> 3072-bit RSA key, ID 4D88605DA7B4752D,
blackbox> created 2021-09-07 (main key ID 58065DE4EAD1E8E1).
blackbox> Passphrase: b
blackbox> gpg-agent[709]: failed to unprotect the secret key: Timeout
blackbox> gpg-agent[709]: failed to read the secret key
blackbox> gpg-agent[709]: command 'PKDECRYPT' failed: Timeout <Pinentry>
blackbox> gpg: decryption failed: No secret key
blackbox> gpg-agent[701]: socket file has been removed - shutting down
blackbox> gpg-agent[701]: gpg-agent (GnuPG) 2.2.27 stopped
blackbox> gpg-agent[709]: socket file has been removed - shutting down
blackbox> gpg-agent[709]: gpg-agent (GnuPG) 2.2.27 stopped
blackbox> make: *** [Makefile:146: confidence] Error 1
```

--- a/pkgs/applications/version-management/blackbox/default.nix
+++ b/pkgs/applications/version-management/blackbox/default.nix
@@ -24,7 +24,8 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ gnupg ];
 
-  doCheck = true;
+  # https://github.com/NixOS/nixpkgs/issues/134445
+  doCheck = !stdenv.isDarwin && stdenv.isx86_64;
 
   checkInputs = [
     expect

